vite.config.js

```diff
@@ -0,0 +1,30 @@
+import copy from 'rollup-plugin-copy';
+import postcss from 'rollup-plugin-postcss';
+import { resolve } from 'path';
+import tailorCe from '@extensionengine/rollup-plugin-tailor-ce';
+import vue from '@vitejs/plugin-vue2';
+
+export default {
+  plugins: [
```

Comment:
Top level ordering of this config object should be, `build`, `resolve`, `plugins`.